distribution/docs/src/main/resources/ManContents.adoc

```diff
@@ -2310,6 +2186,36 @@ The `system.properties` file found under `$DDF_HOME/etc` contains properties tha
 
 By default, this value is set to 128. It is not recommended to set this value extremely high. If unsure, leave this setting at it's default value of 128.
 
+=== Configuring Global System Properties
+The `system.properties` file found under `$DDF_HOME/etc` contains properties that will be made available through system properties at the beginning of Karaf's boot process. After changing any of these properties you will need to restart the system. If you change the 'hostname' property you will also need to configure the certs as described in Configuring {branding} with New Certificates.
```

Comment:
I was borrowing this from the section above it. It is just trying to indicate that these properties will always be available within our system. I can change the wording to say as much.